### PR TITLE
Tutorial for user story M0.1

### PR DESCRIPTION
**Pull request by guarin**
## Changes

- Add tutorial for the new workflow using the docker + platform
- see: [Trigger a Docker Job from the API using a Remote Datasource — lightly 1.2.9 documentation.pdf](https://github.com/lightly-ai/lightly/files/8179177/Trigger.a.Docker.Job.from.the.API.using.a.Remote.Datasource.lightly.1.2.9.documentation.pdf)

### CODE REVIEW

File: docs/source/docker/integration/docker_trigger_from_api.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+To trigger a new job you can click on the schedule run button on the dataset
+overview as shown in the screenshot below:
+
+.. image:: images/schedule-compute-run.png
+
+After clicking on the button you will see a wizzard to configure the the parameters
+for the job.
+
+.. image:: images/schedule-compute-run-config.png
```
Comment (MalteEbner) [inline docs/source/docker/integration/docker_trigger_from_api.rst:97]: You could change the image to the updated one:
https://user-images.githubusercontent.com/20324507/156338271-ea4ee2a7-b3d7-48da-9d3a-76f87c3c0e97.jpeg

File: docs/source/docker/integration/docker_trigger_from_api.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
+.. _integration-docker-trigger-from-api:
+
+Trigger a Docker Job from the API using a Remote Datasource
+===========================================================
+
+Introduction
+------------
+The Lightly workflow can be completely automated by using the Lightly Docker as
```
Comment (MalteEbner) [inline docs/source/docker/integration/docker_trigger_from_api.rst:8]: It is still called "Lightly Docker" here. Why not call it "Lightly compute worker"? Or would you wait with that until we have decided on another newer name?

**Review by MalteEbner — COMMENTED** [review]
In general this looks very good. Great that you could recycle so much from the M0.0 tutorial.

The biggest problem I see is that the config will be update as soon as dev is released, the change is already merged.https://github.com/lightly-ai/lightly-core/pull/1380

File: docs/source/docker/integration/docker_trigger_from_api.rst (hunk 2 above)
Comment (IgorSusmelj) [reply docs/source/docker/integration/docker_trigger_from_api.rst:8]: Let's wait with the renaming and then use find to find all the spots. So far we mostly used the word "docker" in connection. So we just search for all "docker" words :)

File: docs/source/docker/integration/docker_trigger_from_api.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+:ref:`cloud bucket <platform-create-dataset>`.
+
+
+Advantages
+----------
+
+- You can submit jobs through the API, fully automating the Lightly workflow.
+- You can automatically trigger a new job when data is added to your dataset.
+- You no longer require direct access to the Lightly Docker
+- You do not have to restart the Lightly Worker between jobs
```
[the remark this answers is not in the document]
Comment (IgorSusmelj) [reply docs/source/docker/integration/docker_trigger_from_api.rst:22]: Good catch!

**Review by MalteEbner — APPROVED** [review]
LGTM.
Now only testing is missing, then we can release M0.1 ☺️

**Pull request merged**